addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -1329,8 +1334,22 @@ registry.WebsiteAnimate = publicWidget.Widget.extend({
                 this.$target.find('.modal:visible').scrollTop() :
                 this.$scrollingElement.scrollTop();
             const elTop = this._getElementOffsetTop($el[0]) - scrollTop;
-
-            const visible = this.windowsHeight > (elTop + elOffset) && 0 < (elTop + elHeight - elOffset);
+            let visible;
+            const footerEl = el.closest('.o_footer_slideout');
+            const wrapEl = this.$target[0];
+            if (footerEl && wrapEl.classList.contains('o_footer_effect_enable')) {
+                // Since the footer slideout is always in the viewport but not
+                // always displayed, the way to calculate if an element is
+                // visible in the footer is different.
+                const actualScroll = wrapEl.scrollTop + this.windowsHeight;
+                const totalScrollHeight = wrapEl.scrollHeight;
+                const heightFromFooter = this._getElementOffsetTop(el, footerEl);
+                visible = actualScroll >=
+                    totalScrollHeight - heightFromFooter - elHeight + elOffset;
+            } else {
+                visible = this.windowsHeight > (elTop + elOffset) &&
+                    0 < (elTop + elHeight - elOffset);
+            }
```

Comment:
Did you consider using `elementFromPoint` (or something similar?) to code something generic? I don't know if that would be viable or good but maybe?

Reply:
Yes that was my first idea as you can see here: https://github.com/odoo/odoo/commit/b939d9e44a5808241e0932bbdd20af942671af9e I had the comment that was too much changes https://github.com/odoo/odoo/pull/93133#discussion_r900034428 so I did it in a different way

Reply:
Mmh, I guess it's debatable. While it is in general true that we want to keep a diff minimal in stable, it's never true in master if the advantage is to solve a problem generically. In this case, as we will target master as confirmed here https://github.com/odoo/odoo/pull/93133#issuecomment-1172452377, I would have gone for the generic way with `elementFromPoint`. Although, what @bvr-odoo said is also right: you probably compared the positions of the elements while it is not at it final position and the code seems too much for such a small use case to fix...

Ok, as the fix is not critical, let's not waste more time on this and just go with your current solution without `elementFromPoint`, just add a comment like "We decided to handle this case specifically instead of a generic solution using `elementFromPoint` as it is a rare case and the implementation would have been too complicated for such a small use case."